samples/nrf9160/modem_shell/src/rest/rest_shell.c

```diff
@@ -42,8 +42,14 @@ static const char rest_shell_cmd_usage_str[] =
 	"                      (default: 1024 bytes)\n"
 	"  -k, --keepalive,    Keep socket connection alive for upcoming requests\n"
 	"                      towards the ame server (default: false)\n"
```

Comment:
Not modified in this PR, but could correct typo while we're at it: ame -> same

Reply:
Changed